### PR TITLE
Chagelog for 1.6.1

### PR DESCRIPTION
**Pull request by akshayhavile**
This PR for mentioning change log for AKO 1.6.1
@sudswasavi @monotosh-avi @chauhanshubham @swathinsankaran @aaha97 : Please review the changes.

### CODE REVIEW

**Comment by akshayhavile** [comment]
build ako

File: CHANGELOG.md [hunk 1]
```diff
@@ -176,3 +176,30 @@ All notable changes to this project will be documented in this file. The format
...
+ - Support custom port numbers for dedicated and shared virtual services through HostRule CR field.
+ - Support for enabling/disabling non-significant logs through HostRule CRD.
+ - Add autoFQDN to shared virtual services.
+ - Support for Analytical profile for virtual service through HostRule CRD.
+ - Support for Static IP for Shared and Dedicated virtual service through HostRule CRD.
+ - Tenant context support for SE group.
+ - Support for multiple alias FQDNs for a host through HostRule CRD.
+ - Support to configure `Pool Placement Setting` for child ingresses/routes/svclb through AviInfraSetting CRD.
+ - Allow adding custom Pod CIDR to Node mapping via annotation.
+ - Support for `Shared virtual service` fqdn in HostRule CRD.
```
Comment (chauhanshubham) [inline CHANGELOG.md:195]: I think this is covered already and can be removed perhaps.

File: CHANGELOG.md [hunk 2]
```diff
@@ -176,3 +176,30 @@ All notable changes to this project will be documented in this file. The format
...
+
+### Changed:
+- Update `annotation` instead of `status` field of AKO statefulset after avi object deletion through `deleteConfig` flag.
+- AKO will create single HTTP Policyset object at Avi Controller side for all paths of same host.
+
+### Known Issues:
+ - AviInfraSetting CR can not be applied to passthrough ingress/route.
+ - AKO is not updating the ingress status when annotation `passthrough.ako.vmware.com/enabled: "true"` is added to the ingress.
+ - There are issues when shardVS size changed through AviInfra CR or values.yaml. Recommended workflow is to first delete existing config using `deleteConfig` flag as described [here](docs/faq.md#how-do-i-clean-up-all-my-configs) and then change `shardVsSize`.
```
Comment (chauhanshubham) [inline CHANGELOG.md:204]: shardVS size **is** changed through **AviInfraSetting**

**Comment by akshayhavile** [comment]
@chauhanshubham: Review comments addressed

**Review by chauhanshubham — APPROVED** [review]

**Review by monotosh-avi — APPROVED** [review]

**Pull request merged**